tactile:
  use_bluetooth: True
#  bluetooth_device_address: '98:D3:11:FC:3E:27'
  bluetooth_device_address: '98:D3:51:FD:EA:1E'
  serial_port: '/dev/ttyUSB0'
  serial_baud: 115200
  sensor_data_topic: 'tactile_sensor_data'

  sensors:
    - id: 1
      width: 6
      height: 6
      value_size: 10
      frame: 'none'
    - id: 2
      width: 6
      height: 6
      value_size: 10
      frame: 'none'
    - id: 3
      width: 6
      height: 6
      value_size: 10
      frame: 'none'
